azure-devops/azext_devops/dev/boards/work_item.py

```diff
@@ -70,7 +70,7 @@ def create_work_item(work_item_type, title, description=None, assigned_to=None,
             if assigned_to == '':
                 resolved_assigned_to = ''
             else:
-                resolved_assigned_to = _resolve_identity_as_unique_user_id(assigned_to, organization)
+                resolved_assigned_to = resolve_identity_as_display_name(assigned_to, organization)
```

Comment:
Two email Ids resolving to same display name can cause issue. WIT API accepts the email id and name itself lets prefer that approach rather than trying to resolve ourself.

Issue with using display name: 

Command to assign an item to person X,
PS D:\GithubRepos\Work\MS\azure-devops-cli-extension\azure-devops> az boards work-item create --title "title" --type bug --assigned-to "lastname.firstname@**something**.com"

Wit gets assigned to person Y-
ID    Type    Title    Assigned To                        State
----  ------  -------  ---------------------------------  -------
81    Bug     title    Atul Bagga <lastname.firstname@**somethingelse**.com>  New

Reply:
Nice catch!